packages/html/src/chart/tests/chart-tooltip.tsx

```diff
@@ -0,0 +1,30 @@
+
+import { AnimationContainer } from '../../animation-container';
+
+const style = `
+    .k-animation-container {
+        display: inline-block;
+        position: relative;
+        overflow: visible;
+    }
+
+    .k-tooltip {
+        position: relative;
+    }
+`;
+
+
+export default () =>(
+    <>
+        <style>{style}</style>
+        <div id="test-area" className="k-d-grid k-grid-cols-1">
+            <AnimationContainer className="k-chart-tooltip-wrapper ">
+                <div className="k-popup">
```

Comment:
you can re-use the `<Popup>` component here.

Reply:
Using the <Popup> component does not suggest the option to set className ("k-chart-tooltip-wrapper") to the AnimationContainer.